### PR TITLE
Modify translation of PRONOM signature to regex #98

### PR DESCRIPTION
**Pull request by mbhopton**
Prevent regex where max repetitions is less than min repetitions. I've set maxoffset to None which means it is not added to the regex. An alternative would be to set it to offset.

I tested with a single PUID since I get an URL error when trying to do fido-prepare on the whole file, leading to the need to prevent the "has_priority_over" check when a puid_filter is used.

Also correct some truth tests "is not None".

### CODE REVIEW

**Comment by mistydemeo** [comment]
> I get an URL error when trying to do fido-prepare on the whole file

This should be fixed by #101.

**Comment by jhsimpson** [comment]
[Fido 1.3.6 release](https://trello.com/c/eSXqehAC/21-fido-136-release)

**Comment by sevein** [comment]
Here's a diff of the generated `fido/conf/formats-v90.xml`. @mbhopton, your changes seem to be working great! I've verified that the new regexes are correct:

```diff
diff --git a/fido/conf/formats-v90.xml b/fido/conf/formats-v90.xml
index cf5f5ee..fd9e36d 100644
@@ -41409,7 +41409,7 @@ Operational Pattern &#8220;OP-Atom&#8221; is intended to satisfy the demands of
       <pattern>
         <position>BOF</position>
         <pronom_pattern>66726565{12}69637066</pronom_pattern>
-        <regex>(?s)\A.{4,0}free.{12}icpf</regex>
+        <regex>(?s)\A.{4}free.{12}icpf</regex>
       </pattern>
     </signature>
     <details>
@@ -41439,7 +41439,7 @@ Operational Pattern &#8220;OP-Atom&#8221; is intended to satisfy the demands of
       <pattern>
         <position>BOF</position>
         <pronom_pattern>6E693100</pronom_pattern>
-        <regex>(?s)\A.{344,0}ni1\x00</regex>
+        <regex>(?s)\A.{344}ni1\x00</regex>
       </pattern>
     </signature>
     <details>
```

Full diff: https://gist.github.com/sevein/83ba5152d4d65704f7693d7750c12492.

**Comment by sevein** [comment]
This closes #98.
